dev-tools/docker/docker-compose.yml

```
@@ -3,17 +3,19 @@ version: "3.4"
 services:
 
 ### container administration helper Container #######################################
+  # This is not meant to use in prod! only in local env for newbies
   # to get the password (hashed): (also remember to escape $ with anotehr $)
```

```suggestion
  # To get the password (hashed): (also remember to escape $ with another $)
```

To differentiate this comment from the previous